pkg/rsyslog/rsyslog.conf

```diff
@@ -35,17 +35,6 @@ main_queue(
   queue.maxFileSize="100m"
 )
 
-template(name="jsonOutput" type="list" option.jsonf="on") {
-  property(outname="source" name="$!source" format="jsonf")
-  property(outname="level" name="$!level" format="jsonf")
-  property(outname="msg" name="$!msg" format="jsonf")
-  property(outname="file" name="$!file" format="jsonf")
-  property(outname="func" name="$!func" format="jsonf")
-  property(outname="time" name="$!time" format="jsonf")
-  property(outname="pid" name="$!pid" format="jsonf")
-  property(outname="partition" name="$/IMGP" format="jsonf")
-}
-
```

Comment:
Hey @gkodali-zededa -- what's the plan here? Are you moving towards getting rid of rsyslog altogether since we won't be using any of its parsing capabilities?

Reply:
@rvs Since these incoming json message already have all the information (that we try and add in the configuration above), this is no longer required. We should also allow free form json messages created by json services and not restrict the fields in them.